app/src/main/java/com/sunlightlabs/congress/services/ProPublica.java

```diff
@@ -36,7 +36,6 @@
 
     public static TimeZone CONGRESS_TIMEZONE = TimeZone.getTimeZone("America/New_York");
     public static String dateOnlyFormat = "yyyy-MM-dd";
-    public static String timeOnlyFormat = "hh:mm:ss";
```

Comment:
This is unused. And when time formats are used for the Pro Publica API, we should be using `HH:mm:ss` to use an hour of `00-24`. There was a bug where votes at 12pm ET were being parsed as 12am ET (and thus 9pm PT the previous day).